x-pack/plugins/lens/public/xy_visualization_plugin/xy_config_panel.tsx

```diff
@@ -66,14 +68,22 @@ const positionIcons = [
   },
 ];
 
-export function XYConfigPanel(props: VisualizationProps<State>) {
-  const { state, datasource, setState } = props;
+export const XYConfigPanel = injectI18n((props: VisualizationProps<State> & InjectedIntlProps) => {
```

Comment:
I know it's still used in a lot of places but `injectI18n` is deprecated and should not be used anymore (reference https://github.com/elastic/kibana/issues/27968 ). I forgot to mention this when we talked about it offline, sorry.

The linked issue describes how it should be done.